assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+                ["N59d14m39.64", "E017d38m02.86"],
+                ["N59d14m44.29", "E017d37m56.62"],
+                ["N59d14m49.10", "E017d37m50.86"],
+                ["N59d14m54.05", "E017d37m45.60"],
+                ["N59d14m59.14", "E017d37m40.85"],
+                ["N59d15m04.35", "E017d37m36.63"],
+                ["N59d15m09.66", "E017d37m32.94"],
+                ["N59d15m15.07", "E017d37m29.81"],
+                ["N59d15m20.55", "E017d37m27.23"],
+                ["N59d15m26.10", "E017d37m25.21"],
+                ["N59d15m31.69", "E017d37m23.76"],
+                ["N59d15m37.31", "E017d37m22.89"],
+                ["N59d15m42.95", "E017d37m22.59"],
+                ["N59d15m48.59", "E017d37m22.87"],
+                ["N59d15m54.21", "E017d37m23.73"],
+                ["N59d15m59.80", "E017d37m25.15"],
+                ["N59d16m05.35", "E017d37m27.15"],
+                ["N59d16m10.83", "E017d37m29.72"],
+                ["N59d16m16.24", "E017d37m32.84"],
+                ["N59d16m21.56", "E017d37m36.51"],
+                ["N59d16m26.77", "E017d37m40.71"],
+                ["N59d16m31.87", "E017d37m45.45"],
+                ["N59d16m36.82", "E017d37m50.70"],
+                ["N59d16m41.64", "E017d37m56.45"],
+                ["N59d16m46.29", "E017d38m02.69"],
+                ["N59d16m50.76", "E017d38m09.39"],
+                ["N59d16m55.05", "E017d38m16.54"],
+                ["N59d16m59.15", "E017d38m24.13"],
+                ["N59d17m03.03", "E017d38m32.12"],
+                ["N59d17m06.70", "E017d38m40.49"],
+                ["N59d17m10.13", "E017d38m49.24"],
+                ["N59d17m13.33", "E017d38m58.32"],
+                ["N59d17m16.28", "E017d39m07.71"],
+                ["N59d17m18.97", "E017d39m17.40"],
+                ["N59d17m21.40", "E017d39m27.35"],
+                ["N59d17m23.56", "E017d39m37.53"],
+                ["N59d17m25.45", "E017d39m47.92"],
+                ["N59d17m27.05", "E017d39m58.49"],
+                ["N59d17m28.37", "E017d40m09.21"],
+                ["N59d17m29.40", "E017d40m20.05"],
+                ["N59d17m30.13", "E017d40m30.98"],
+                ["N59d17m30.58", "E017d40m41.97"],
+                ["N59d17m30.72", "E017d40m53.00"]
+            ]
+        }
+    ],
+    "_comment": "Source runways: https://www.aro.lfv.se/Editorial/View/7952/ES_AD_2_ESSA_en",
+    "runways": [
+        {
+            "name": ["01L", "19R"],
+            "end":    [["N59d38m14.11", "E017d54m47.60", "98.6ft"], ["N59d39m59.04", "E017d55m25.56", "118.2ft"]],
+            "ils": [true, true]
+        },
+        {
+            "name": ["01R","19L"],
+            "end": [["N59d37m35.03", "E017d57m02.67", "138.0ft"], ["N59d38m54.49", "E017d57m31.48", "98.9ft"]],
+            "ils": [true, true]
+        },
+        {
+            "name": ["08", "26"],
+            "end": [["N59d39m30.31", "E017d56m10.08", "108.0ft"], ["N59d39m50.03", "E017d58m44.96", "124.8ft"]],
+            "ils": [false, true]
```

Comment:
According to docs runway 08 has localizer only, no glideslope. Is there any possibility to implement this in the sim? Maybe there's no point? Should I just keep it at false and be satisfied with that?